controllers/oauthAuthentication.js

```diff
@@ -2,18 +2,25 @@
  * Helper to get an access token from France Connect.
  * @see @link{ https://partenaires.franceconnect.gouv.fr/fcp/fournisseur-service# }
  */
+import Joi from '@hapi/joi';
 import querystring from 'querystring';
 import { httpClient } from '../helpers/httpClient';
 import config from '../config';
-import { getAcrFromIdToken } from '../helpers/utils';
+import {
+  getAcrFromIdToken,
+  QUERY_ERROR_REGEX,
+  QUERY_CODE_REGEX,
+} from '../helpers/utils';
 
 /**
  * Format the url use in the redirection call
  * to the France Connect Authorization and logout API endpoint.
  * @see @link{ https://partenaires.franceconnect.gouv.fr/fcp/fournisseur-service# }
  */
 export const oauthLoginAuthorize = (req, res) => {
-  const eidasQueryString = req.body.eidasLevel ? `&acr_values=${req.body.eidasLevel}` : '';
+  const eidasQueryString = req.body.eidasLevel
```

Comment:
plz used this project linter config not you local config.

Reply:
![capture](https://user-images.githubusercontent.com/57352461/70051745-f3272b00-15d1-11ea-9bd5-f6641a6dafb0.png)

Déjà le cas

Reply:
Exactly what I show you previously! the linter there is that of the project not of my IDE, I use eslint project controls.